python/examples/volume/org.xen.xapi.storage.simple-file/sr.py

```diff
@@ -73,10 +73,10 @@ def attach(self, dbg, configuration):
             urllib.urlencode(configuration, True),
             None))
 
-    def detach(self, dbg, configuration):
+    def detach(self, dbg, sr):
         """
-        [attach configuration]: attaches the SR to the local host. Once an SR is
-        attached then volumes may be manipulated.
+        [detach sr]: detaches the SR, clearing up any associated resources.
+        Once the SR is detached then volumes may not be manipulated.
```

Comment:
What does "may" mean in this context? I would expect either "must" or "can't".

Reply:
The comment is, as it should always have been, a copy of the one here https://github.com/xapi-project/xapi-storage/blob/master/generator/lib/control.ml#L413 on the IDL definition.

Reply:
but it does indeed mean `cannot`

Reply:
different context of may which can mean either optional or you cannot, English is not a very precise language

Reply:
For technical documentation I like what the language used in RFCs: http://microformats.org/wiki/rfc-2119

Reply:
Agreed, but, I didn't write the original comment, that would probably be JonL. If we change it we should change both. I was just making the example consistent with the IDL.